website/docs/r/firewall.html.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: "gridscale"
+page_title: "gridscale: firewall"
+sidebar_current: "docs-gridscale-resource-firewall"
+description: |-
+  Manages a firewall in gridscale.
+---
+
+# gridscale_firewall
+
+Provides a firewall resource. This can be used to create, modify and delete firewalls.
+
+## Example Usage
+
+```terraform
+resource "gridscale_firewall" "foo" {
```

Comment:
Plz, would you give an example with a nice naming instead of `newname`  and `comment = "testv6"`!!

Reply:
They are just examples and still easily understandable. I don't see why we should pay much attention to naming.

Reply:
I know they are just examples. However, these examples are shown to customers in our Terraform docs. Anyhow, would you fix at least my other comment? Thanks